### PR TITLE
GDS build server

### PR DESCRIPTION
**Pull request by emileswarts**
Documentation for understanding and debugging internal backups

Screenshot 2019-05-29 at 16 08 03

### CODE REVIEW

File: source/documentation/gds_on_site_backups.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The script that does the backups is located in: 
+
+`/root/Scripts/govwifi-backup.sh`
+
+You can ssh into this server:
+
+`ssh -i ~/.ssh/bob -vvvv bob@ah-govwf-d-01.dmz.gds`
+
+You need to be on a VPN to gain access to this server.
```
Comment (paroxp) [inline source/documentation/gds_on_site_backups.md]:
```suggestion
You need to be on a GovWifi specific VPN to gain access to this server.
```

File: source/documentation/gds_on_site_backups.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Please speak to the security team to get you set up on this.
```
Comment (paroxp) [inline source/documentation/gds_on_site_backups.md]: I feel like we may need to onboard individuals to that server. I'm not certain if this responsibility lies with IT or the team. Worth clarifying?

File: source/documentation/gds_on_site_backups.md (hunk 1 above)
Comment (emileswarts) [reply source/documentation/gds_on_site_backups.md]: 👍

File: source/documentation/gds_on_site_backups.md (hunk 2 above)
Comment (paroxp) [reply source/documentation/gds_on_site_backups.md]: Sorry, what I meant is that VPN is not enough. There needs to be the `useradd -m -G sudo USERNAME` action performed by someone with existing sudo access.

**Pull request merged**